resources/extruders/hellbot_magna_dual_extruder_2.def.json

```
@@ -0,0 +1,16 @@
+ {
+    "id": "hellbot_magna_dual_extruder_2",
+    "version": 2,
+    "name": "Extruder 2",
+    "inherits": "fdmextruder",
+    "metadata": {
+        "machine": "hellbot_magna_dual",
+        "position": "0"
```

This should be position `1`
```suggestion
        "position": "1"
```